playwright/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM mcr.microsoft.com/playwright:v1.18.1-focal-amd64
+
+ARG DEBIAN_FRONTEND=noninteractive
+ARG TZ=America/Los_Angeles
```

Comment:
Do you know if we need these two args? I'd imagine that since we are using `-y` with `apt-get install` then we don't need `DEBIAN_FRONTEND=noninteractive`; I wonder about setting the timezone to LA.

Reply:
Removed!

Reply:
> I believe you can just use `ARG BUILDARCH` here, which will give you access to a correct `${BUILDARCH} later in the file.

This fails for me locally unless I also supply an appropriate `--build-arg` .